src/components/WwwFrame/TheHeader.vue

```diff
@@ -340,41 +340,65 @@
 								}"
 							>
 								<router-link
+									v-if="enableBasketExperiment"
```

Comment:
Does this new basket icon/button also need the `data-testid="header-basket"` attribute? Please check how robot + cypress tests are using that test id.